src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/TreeView/GroupedByTargetTreeViewProviderTests.cs

```diff
@@ -32,7 +33,7 @@ public void GrouppedByTargetTreeViewProvider_WhenEmptySnapshot_ShouldJustUpdateD
 
             // Act
             var provider = new GroupedByTargetTreeViewProvider(treeServices, treeViewModelFactory, commonServices);
-            var resultTree = provider.BuildTree(dependenciesRoot, snapshot);
+            var resultTree = await provider.BuildTreeAsync(dependenciesRoot, snapshot);
```

Comment:
I thought we ran the analyzer for tests too but actually [we don't](https://github.com/dotnet/project-system/blob/master/build/Rulesets/NonShippingProjectRules.ruleset#L8) - so ignore the comment.